multinet/api/tasks/__init__.py

```diff
@@ -0,0 +1,61 @@
+import celery
+from celery.utils.log import get_task_logger
+
+from multinet.api.models import Task
+
+logger = get_task_logger(__name__)
+
+
+class MultinetCeleryTask(celery.Task):
```

Comment:
What's the advantage of putting this class in the `__init__.py` file instead of a dedicated file with perhaps a more descriptive name of what's in it?

I don't necessarily object to this being here, just curious to know if there was a particular reason to put this class here.

Reply:
There's no huge advantage, just that there's no other files at this level, and the class can be imported with `from multinet.api.tasks import MultinetCeleryTask` without needing to be imported in this file manually (if it were defined in another file). But yeah, there's no real advantage or disadvantage. If I only have one or two things for a given module, I'll sometimes just place those things in `__init__.py`. It's not uncommon to my knowledge.

Reply:
Fair enough. Thanks for clearing that up.